cpp/part1/connect.cpp

```diff
@@ -10,7 +10,7 @@ int main(int argc, char* argv[]) {
 
   // Add the contact points. These can be either IPs or domain names.
   // You can specify more than one, comma-separated, but you don’t have to - driver will discover other nodes by itself. You should do it if you expect some of your contact points to be down.
-  cass_cluster_set_contact_points(cluster, "172.21.0.2"); // set the IP according to your setup
+  cass_cluster_set_contact_points(cluster, "172.18.0.2"); // set the IP according to your setup
```

Comment:
This seems random
If the IP change on every run, why change it?

Reply:
It's not very important. In the lesson (and in the code comments above) the instructions tell the user to change the IP according to their cluster. 
I wanted it to be the same in Part 1 and Part 2 but it would be changed by the user anyhow so it doesn't matter much.